### PR TITLE
docs: Add wizard getting started documentation

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): fixes https://github.com/unoplatform/uno/issues/4321

## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/get-started-wizard.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Welcome to the Uno Platform
+
+Congratulations, you've just created a new project using the Uno Platform project templates.
+
+* You can give a try to the [Single Page app getting started guide](https://platform.uno/docs/articles/getting-started-tutorial-1.html)
+* Next you can try our [Bug Tracker sample walk-through](https://platform.uno/docs/articles/getting-started-tutorial-2.html)
+* More advanced examples in our [Uno.Samples repository](https://github.com/unoplatform/uno.samples)
```
Comment (sasakrsmanovic) [inline doc/articles/get-started-wizard.md:7]:
```suggestion
* More advanced examples in our [Uno.Samples repository](https://github.com/unoplatform/uno.samples)
* Fork a fully-fledged [Ch9 application and source code ](https://platform.uno/code-samples/#ch9)
```

**Review by sasakrsmanovic — APPROVED** [review]

File: doc/articles/get-started-wizard.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Welcome to the Uno Platform
+
+Congratulations, you've just created a new project using the Uno Platform project templates.
+
+* You can give a try to the [Single Page app getting started guide](https://platform.uno/docs/articles/getting-started-tutorial-1.html)
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:5]:
```suggestion
* You can give the [Single Page app getting started guide](https://platform.uno/docs/articles/getting-started-tutorial-1.html) a try
```

File: doc/articles/get-started-wizard.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+## Common Issues
+The Uno Platform features and support is constantly evolving, but you may encounter issues while building your application.
+
+### XAML Hot Reload for the UWP head may not work properly in Visual Studio 2019 16.7 and earlier
+The [XAML Hot Reload](https://docs.microsoft.com/en-us/visualstudio/xaml-tools/xaml-hot-reload?view=vs-2019) feature [does not work in Visual Studio 2019 16.7](https://developercommunity.visualstudio.com/content/problem/996417/xaml-hot-reload-hot-reload-is-not-working-in-share.html) and earlier for Uno projects. This will be fixed in 16.8 and later.
+
+### The XAML editor shows `The type 'page' does not support direct content` message
+XAML Intellisense [is not working properly](https://developercommunity.visualstudio.com/content/problem/587980/xaml-intellisense-does-not-use-contentpropertyattr.html) in Visual Studio when the active project is not the UWP one. 
+
+To work around this issue, close all XAML editors, open a C# file and selec the UWP in the top left drop-down list of the text editor sector. Once selected, re-open the XAML file.
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:19]:
```suggestion
To work around this issue, close all XAML editors, open a C# file and select 'UWP' in the top left drop-down list of the text editor sector. Once selected, re-open the XAML file.
```

File: doc/articles/get-started-wizard.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+### The XAML editor may be empty
+The XAML editor may be empty when the active editor project is not UWP.
+
+To work around this issue, close all XAML editors, open a C# file and selec the UWP in the top left drop-down list of the text editor sector. Once selected, re-open the XAML file.
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:24]:
```suggestion
To work around this issue, close all XAML editors, open a C# file and select 'UWP' in the top left drop-down list of the text editor sector. Once selected, re-open the XAML file.
```

File: doc/articles/get-started-wizard.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+### The path may be too long
+The project may fail to build is the path of the project is too long.
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:27]:
```suggestion
The project may fail to build if the path of any file in the project is [too long](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation).
```

File: doc/articles/get-started-wizard.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+To work around this issue, either :
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:29]:
```suggestion
To work around this issue, either:
```

File: doc/articles/get-started-wizard.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+- Move the project to a shorter path
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:30]:
```suggestion
- Move the project to a shorter root path, or
```

File: doc/articles/get-started-wizard.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+### `x:Name` variable is not available in code-behind
+Visual Studio 16.7 and earlier versions [do not refresh the intellisense cache ](https://developercommunity.visualstudio.com/content/problem/588021/the-compile-itemgroup-intellisense-cache-is-not-re.html)properly, causing variables to be incorrectly defined.
+
+To fix this issue, build your project once, close the solution and reopen it.
+
+### "Missing value for TargetPlatformWinMDLocation property" when adding a project reference
+This issue is caused by [VS 2019 support for SDK-Style projects](https://developercommunity.visualstudio.com/content/problem/1170010/missing-value-for-targetplatformwinmdlocation-prop.html).
+
+To add a reference change the list
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:44]: Incomplete?

File: doc/articles/get-started-wizard.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+Visual Studio 16.7 and earlier versions [do not refresh the intellisense cache ](https://developercommunity.visualstudio.com/content/problem/588021/the-compile-itemgroup-intellisense-cache-is-not-re.html)properly, causing variables to be incorrectly defined.
+
+To fix this issue, build your project once, close the solution and reopen it.
+
+### "Missing value for TargetPlatformWinMDLocation property" when adding a project reference
+This issue is caused by [VS 2019 support for SDK-Style projects](https://developercommunity.visualstudio.com/content/problem/1170010/missing-value-for-targetplatformwinmdlocation-prop.html).
+
+To add a reference change the list 
+
+### Build fails `error : Error reading response`
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:46]:
```suggestion
### Build fails with `error : Error reading response`
```

File: doc/articles/get-started-wizard.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+In general, this error happens when the XAML parser detects a syntax error. Fixing the error generally fixes the build.
+
+This error may happen occasionally without any explicit error message, rebuilding the project may fix the issue.
+
+### Event cannot be added
+
+Events [cannot be automatically](https://github.com/unoplatform/uno/issues/1348#issuecomment-520300471) added using the XAML editor. 
+
+A workaround is to use the [`x:Bind` to events feature](https://github.com/unoplatform/uno/blob/master/doc/articles/features/windows-ui-xaml-xbind.md#examples) and manually create a method in the code-behind.
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:55]: maybe explain the advantage of this approach (that it doesn't require remembering a complicated method signature?)

File: doc/articles/get-started-wizard.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+This issue is caused by [VS 2019 support for SDK-Style projects](https://developercommunity.visualstudio.com/content/problem/1170010/missing-value-for-targetplatformwinmdlocation-prop.html).
+
+To add a reference change the list 
+
+### Build fails `error : Error reading response`
+In general, this error happens when the XAML parser detects a syntax error. Fixing the error generally fixes the build.
+
+This error may happen occasionally without any explicit error message, rebuilding the project may fix the issue.
+
+### Event cannot be added
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:51]:
```suggestion
### Event handler cannot be added automatically
```

File: doc/articles/get-started-wizard.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+
+Events [cannot be automatically](https://github.com/unoplatform/uno/issues/1348#issuecomment-520300471) added using the XAML editor.
```
Comment (davidjohnoliver) [inline doc/articles/get-started-wizard.md:53]:
```suggestion
Event handlers [cannot be automatically](https://github.com/unoplatform/uno/issues/1348#issuecomment-520300471) added using the XAML editor. 
```

**Review by davidjohnoliver — APPROVED** [review]

**Review by agneszitte-nventive — APPROVED** [review]

**Pull request merged**